From foo@baz Fri Jan 18 09:08:25 CET 2019
Date: Fri, 18 Jan 2019 09:08:25 +0100
To: Greg KH <gregkh@linuxfoundation.org>
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Subject: test: runtime test

If this gets noticed by the build/test infrastucture, then it is working properly.

Don't merge this, for obvious reasons...

diff --git a/init/main.c b/init/main.c
index ee147103ba1b..1a3d722f25c2 100644
--- a/init/main.c
+++ b/init/main.c
@@ -1086,6 +1086,12 @@ static int __ref kernel_init(void *unused)
 
 	rcu_end_inkernel_boot();
 
+	/*
+	 * If you notice this, your test infrastructure succeeded in finding a
+	 * failure, congratulations
+	 */
+	panic("boot canary hit");
+
 	if (ramdisk_execute_command) {
 		ret = run_init_process(ramdisk_execute_command);
 		if (!ret)
